src/main/java/frc/team4909/robot/subsystems/climber/commands/SetStiltPosition.java

```diff
@@ -21,79 +21,61 @@ protected void initialize() {
 
      @Override
     protected void execute() {
-        double moveSpeedBoth = -Robot.climberGamepad.getThresholdAxis(BionicF310.LY)
+        double moveSpeed = -Robot.manipulatorGamepad.getThresholdAxis(BionicF310.LY) // elevator speed for manipulator gamepad
+        * RobotConstants.elevatorSpeedMultiplier;
+        
+        double moveStiltSpeed = -Robot.climberGamepad.getThresholdAxis(BionicF310.RY) //Stilt Speed
         * RobotConstants.climbSpeedMultiplier;
-
-        double moveSpeed = -Robot.manipulatorGamepad.getThresholdAxis(BionicF310.LY)
+        
+        double moveElevatorSpeed = -Robot.climberGamepad.getThresholdAxis(BionicF310.LY) // elevator speed for climber gamepad
         * RobotConstants.elevatorSpeedMultiplier;
-
-        double moveDrivePosSpeed = -Robot.driverGamepad.getThresholdAxis(BionicF310.RT) 
+        
+        double driveStiltPosSpeed = -Robot.driverGamepad.getThresholdAxis(BionicF310.RT)  // Stilt drive forward
         * RobotConstants.climberDriveSpeedManual;
         
-        double moveDriveNegSpeed = Robot.driverGamepad.getThresholdAxis(BionicF310.LT) 
+        double driveStiltNegSpeed = Robot.driverGamepad.getThresholdAxis(BionicF310.LT)  //Stilt drive backwards
         * RobotConstants.climberDriveSpeedManual;
 
-        double moveElevatorSpeed = -Robot.climberGamepad.getThresholdAxis(BionicF310.RY)
-        * RobotConstants.elevatorSpeedMultiplier;
-
-        // Set constant slow drive speed for wheels
-        Robot.climberSubsystem.setStiltsDriveSpeed(RobotConstants.climberDriveSpeedAuto);
+        Robot.climberSubsystem.setStiltsDriveSpeed(RobotConstants.climberDriveSpeedAuto); // Stilts via climnber
         
 
-         if(moveSpeed==0 && moveSpeedBoth == 0 && moveElevatorSpeed == 0){
+         if(moveSpeed==0 && moveStiltSpeed == 0 && moveElevatorSpeed == 0){
              SmartDashboard.putString("thisblock", "1");
              Robot.climberSubsystem.setPosition(Robot.climberSubsystem.holdingStiltsPosition);
              Robot.elevatorSubsystem.setPosition(Robot.elevatorSubsystem.holdingPosition);
          }
-         else if (moveSpeed != 0 && moveSpeedBoth == 0 && moveElevatorSpeed == 0 ) { // If Y-stick value is not moving, HOLD position
+         else if (moveSpeed != 0 && moveStiltSpeed == 0 && moveElevatorSpeed == 0 ) { // Elevator via manipulator gamepad
             SmartDashboard.putString("thisblock", "2");
             Robot.elevatorSubsystem.setSpeed(moveSpeed);
             Robot.elevatorSubsystem.holdingPosition = Robot.elevatorSubsystem.getPosition();
          }
-         else if(moveSpeed == 0 && moveSpeedBoth != 0  && moveElevatorSpeed == 0){
-            SmartDashboard.putString("thisblock", "3 here "+ moveSpeedBoth);
-
-            // Elevator Drum is 1.3" Diameter, C = PI * D = Math.PI * 1.3
-            // Stilts pinion gear Pitch Diameter is 1.1" which is the circumference
```

Comment:
Critical comment got lost!!!! Needs to be mentioned or referenced in every place where that 1.1 and 1.3 are used.